src/ruby/pb/test/server.rb

```diff
@@ -170,26 +192,32 @@ def empty_call(_empty, _call)
   end
 
   def unary_call(simple_req, _call)
+    maybe_echo_metadata(_call)
+    maybe_echo_status_and_message(simple_req)
     req_size = simple_req.response_size
     SimpleResponse.new(payload: Payload.new(type: :COMPRESSABLE,
                                             body: nulls(req_size)))
   end
 
   def streaming_input_call(call)
+    maybe_echo_metadata(call)
```

Comment:
do these need to be on streaming input/output? Just caught since it looks like the tests are only for unary/bidi